### ISSUE TITLE
plugin breaks diff mappings

### ISSUE THREAD

**Issue by smutch** [goldfeld/vim-seek#20]
With vim-seek loaded (via vundle), diff mappings such as `do` and `dp` fail to work in diff mode.

**Comment by goldfeld** [comment]
Yes, I'm sorry it's still undocumented, the switch to get back the native diff mappings is `let g:seek_use_vanilla_binds_in_diffmode = 0`

Vanilla here meaning native. Let me know if it works, it's been a while. I plan to do a major refactor on seek soon (without breaking backwards compatibility of course.)

**Comment by smutch** [comment]
Thanks for looking into this...  The above switch seemed to work fine in restoring the diff mappings.

Thanks for a great plugin!
